### PR TITLE
Docs: Migrate tutorials to Killercoda

### LINKED ISSUE #5385: Docs: Find a replacement for Katacoda.com for tutorials
Katacoda.com is officially being shutdown by 15th June: https://www.oreilly.com/online-learning/leveraging-katacoda-technology.html 

We should find an alternative provider to which we can move or (temporarily) remove Katacoda tutorials from docs.

### PR DESCRIPTION
**Pull request by matej-g**
* [ ] I added CHANGELOG entry for this change.
* [X] Change is not relevant to the end user.

## Changes

Fixes #5385.

After we had to temporarily remove Katacoda tutorials in https://github.com/thanos-io/thanos/pull/5454, we can now add them back after we have successfully migrated to Killercoda :tada: 

Tutorials are available under https://killercoda.com/thanos

The tutorial definitions have now been moved to https://github.com/thanos-io/tutorials.

## Verification
We manually went through the tutorials.

### CODE REVIEW

**Review by douglascamata — DISMISSED** [review]
🚀 :shipit: 🌔

File: docs/getting-started.md [hunk 1]
```diff
@@ -79,6 +79,7 @@ If you want to add yourself to this list, let us know!
 
 ## Deploying Thanos
 
+* [WIP] Detailed, free, in-browser interactive tutorial [as Katacoda Thanos Course](https://killercoda.com/thanos/)
```
Comment (GiedriusS) [inline docs/getting-started.md:82]: So Killercode or Katacoda? :smile: I think this needs to be:
```suggestion
* [WIP] Detailed, free, in-browser interactive tutorial [as Killercoda Thanos Course](https://killercoda.com/thanos/)
```

**Review by GiedriusS — APPROVED** [review]
One suggestion

File: docs/getting-started.md (hunk 1 above)
Comment (matej-g) [reply docs/getting-started.md:82]: I had to leave at least one old reference to make sure reviewers are paying attention :detective:, thanks!

**Comment by matej-g** [comment]
Skipping tests since it's just doc update.

**Pull request merged**